build.gradle.kts

```
@@ -27,12 +42,14 @@ dependencies {
 
     implementation(platform("org.openrewrite:rewrite-bom:${rewriteVersion}"))
     implementation("org.openrewrite:rewrite-java")
+    implementation("org.openrewrite:rewrite-maven")
     implementation("org.openrewrite.recipe:rewrite-java-dependencies:${rewriteVersion}")
     implementation("org.openrewrite.recipe:rewrite-static-analysis:${rewriteVersion}")
     runtimeOnly("org.openrewrite:rewrite-java-17")
 
     implementation("org.apache.logging.log4j:log4j-core:2.+")
     implementation("org.slf4j:slf4j-api:2.+")
+    implementation("org.apache.logging.log4j:log4j-converter-config:0.3.0-SNAPSHOT")
```

Indeed likely best to merge this once released; thanks for creating the recipe before then!
```suggestion
    implementation("org.apache.logging.log4j:log4j-converter-config:latest.release")
```